modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/CommandProcessor.java

```diff
@@ -939,11 +940,7 @@ else if (cmdH2 instanceof GridSqlAlterTableDropColumn) {
                 else {
                     assert tbl.rowDescriptor() != null;
 
-                    GridCacheContext cctx = tbl.cacheContext();
```

Comment:
For not started caches we have a special structure GridCacheContextInfo which can provide base information about the cache without starting cache itself. I see that you already use it in a new introduced method below. For consistency with the other parts I propose to introduce the `mvccEnabled` method in the GridCacheContextInfo and use it instead of current GridCacheContext

Reply:
Agree with proposal. Please check out last commit.